lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -364,8 +364,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :type => Array,
           :allowed_from_server => false,
           :description => <<-DESCRIPTION
-An array of ActiveSupport custom events names to subscribe to and provide
-instrumentation for. For example,
+An array of ActiveSupport custom event names to subscribe to instrument. For example,
```

Comment:
Whoops, I wrote this slightly differently than intended:
```suggestion
An array of ActiveSupport custom event names to subscribe and instrument. For example,
```